src/int/cmp.rs

```diff
@@ -0,0 +1,265 @@
+//! [`Int`] comparisons.
+//!
+//! By default, these are all constant-time and use the `subtle` crate.
```

Comment:
The `Const*` stuff doesn't use `subtle`. If this comment is copypasta from `uint/cmp.rs`, the comment there is probably out-of-date.

Reply:
Done. Fixed it for `uint::cmp` as well.